app/routes/users/UserProfileRoute.js

```diff
@@ -29,21 +29,20 @@ const loadData = (
     match: {
       params: { username },
     },
+    isMe,
   },
   dispatch
-) => {
-  return dispatch(fetchUser(username)).then((action) =>
-    Promise.all([
-      dispatch(fetchPrevious()),
-      dispatch(fetchUpcoming()),
-      dispatch(fetchAllWithType('klasse')),
-    ])
-  );
-  // TODO: re-enable when the user feed is fixed:
-  // .then(action =>
-  //   dispatch(fetchUserFeed(action.payload.result))
-  //  );
-};
+) =>
+  Promise.all([
+    dispatch(fetchAllWithType('klasse')),
+    isMe && dispatch(fetchPrevious()),
+    isMe && dispatch(fetchUpcoming()),
+    isMe || dispatch(fetchUser(username)),
```

Comment:
Why can we ignore fetching the user data?

Reply:
Since it is the same endpoint as we use when fetching the user info for the logged in user. Might be na issue when the user changes some settings tho., so will have to look at that.

Reply:
Yeah, the USER.UPDATE.SUCCESS payload is reduced correctly into the state, so this works as intended. :smile:

Reply:
Nice! :100: